packages/shorebird_cli/lib/src/executables/gradlew.dart

```diff
@@ -93,7 +93,26 @@ class Gradlew {
       if (match != null) {
         final variant = match.group(1)!;
         if (!variant.toLowerCase().endsWith('test')) {
-          variants.add(variant[0].toLowerCase() + variant.substring(1));
+          // Gradle flavor name transformation seems to work following these two
+          // conditions:
+          //  - Flavor starts with at least two capital latter, use as is
+          //  - Otherwise, transform in camel case
```

Comment:
```suggestion
          // Gradle flavor name transformation seems to work with the following rules:
          //  - If the flavor starts with at least two capital letters, use as is
          //  - Otherwise, transform to camel case
```